telemetry/telemetry_record.py

```diff
@@ -73,15 +100,48 @@ def move_wpr_files(name_schema):
     """
     remote_data_path = os.path.join(CHROMIUM_SRC, 'tools/perf/page_sets/data/',
             name_schema)
-    local_path = os.path.join(os.getcwd(), 'data/')
+    local_path = os.path.join(os.getcwd(), 'data/wpr_source')
     p = Popen('mv {0} {1}'.format(remote_data_path, local_path), shell=True)
 
+def make_modified_json(num_urls):
+    """Creates *.json files to point to the right archive files
+
+    Located in ~/src/tools/perf/page_sets/data
+    :param num_urls: int for the number of target urls
+    TODO: add sha1?
+    """
+    json_path = '/home/jamshed/src/tools/perf/page_sets/data/'
+    json_template = ('{{'
+            '"description": "Describes the Web Page Replay archives for a user '
+            'story set. Dont edit by hand! Use record_wpr for updating.", '
+            '"archives": {{ "url{0}_page_set_000.wpr": ["url{0}"]}}}}\n')
+    for i in range(num_urls):
+        modified_name = '999999' + str(i)
+        file_name = 'url{0}_page_set.json'
+        with open('{0}{1}'.format(json_path,
+            file_name.format(modified_name)), 'wb') as f:
+            f.write(json_template.format(modified_name))
+
 def modify_wpr():
-    """Sets cacheable objects delay time to 0, creates  pc.wpr files"""
-    wpr_directory = os.path.join(os.getcwd(), 'data/')
+    """Sets cacheable objects delay time to 0, creates pc.wpr files
+
+    Note: *.pc.wpr files are converted to 999999 + str(url name), which lets
+    them be treated like regular urls. Just know that six 9's indicate a
+    modified wpr file
+    TODO: Fix the 999999 notation
+    """
+    wpr_directory = os.path.join(os.getcwd(), 'data/wpr_source')
     wpr_path = os.path.join(CHROMIUM_SRC,
             'third_party/webpagereplay/modify_wpr_delays.py')
     p = Popen('python {0} {1}'.format(wpr_path, wpr_directory), shell=True)
+    p.wait()
+    pc_files = filter(lambda x: 'pc' in x, os.listdir(wpr_directory))
+    for pc_file in pc_files:
+        new_file = re.sub(r'url(\d+)*', r'url999999\1', pc_file).replace('.pc',
+                '')
+        p = Popen('mv {0}/{1} {0}/{2}'.format(wpr_directory, pc_file, new_file),
```

Comment:
Do you know if shutil sleeps/waits for the process to finish before moving on? When moving a lot of files, this can't move forward until the move is complete.